Makefile

```diff
@@ -109,11 +109,10 @@ cobertura:
 		$(GOCOVER_COBERTURA) > $(GO_TEST_OUTPUT)/cobertura-coverage.xml
 
 # Ensure a PR is ready for review.
-reviewable: generate lint
-	@go mod tidy
+reviewable: generate lint go.vendor.tidy
 
 # Ensure branch is clean.
-check-diff: reviewable
+check-diff: generate go.vendor.tidy
```

Comment:
One thing to note here is that theoretically at least running `golangci-lint` can result in changes. It has a `--fix` flag (that we do not enable by default) that allows it to automatically fix a small set of the linter rules it knows.

Reply:
That's a great point. I had considered it and exactly because `--fix` is not enabled in the config or CLI, I thought this could be fine to drop from `check-diff`.

On the other hand, this can be reverted back to ensure we always run `golang-ci` variant of `build/` submodule at least somewhere on the CI (with a very minimal time penalty).